### DISCUSSION TITLE
My index.html file is not linking to my css file

### DISCUSSION THREAD

**Discussion by 2div** [community/community#23604] [New to GitHub]
<p>Dear all,<br>
i am new to github , i have design a small templetate and uploaded to github. When i click on the website link below , it show that my css style not applied on the html file. i tried to solve it but nothing work.</p>
<p><a href="https://2div.github.io/html_css_temp1/" class="onebox" target="_blank" rel="noopener nofollow ugc">https://2div.github.io/html_css_temp1/</a></p>
<p>please advise how to handle this problem… thanks</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/d/bd4694f0f283927c4702caac8607df5b276be575.jpeg" data-download-href="/uploads/short-url/r0psGIS3uvwG3lDU8SgU8fLMAKN.jpeg?dl=1" title="issue" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/96177680/181078167-1f39a964-547d-42ab-85f2-e0a09fadf9f0.jpeg" alt="issue" data-base62-sha1="r0psGIS3uvwG3lDU8SgU8fLMAKN" width="690" height="349" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/d/bd4694f0f283927c4702caac8607df5b276be575_2_690x349.jpeg, https://aws1.discourse-cdn.com/github/original/3X/b/d/bd4694f0f283927c4702caac8607df5b276be575.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/b/d/bd4694f0f283927c4702caac8607df5b276be575.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/d/bd4694f0f283927c4702caac8607df5b276be575_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">issue</span><span class="informations">1003×508 72.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment, marked as answer]
<p>you have a capitalized folder name, <code>CSS</code> not <code>css</code>, so change that at your link</p>

**Comment by 2div** [comment]
<p>finally it works … thank you <a class="mention" href="/jdevstatic">@jdevstatic</a> very much for your quick reply and advise …</p>
